Makefile

```diff
@@ -43,14 +43,8 @@ CTR_CMD_PUSH_OPTIONS ?=
 
 GENERAL_TAGS := include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo
 GPU_TAGS :=
-ifeq ($(shell ldconfig -p | grep -q libnvml_injection.so && echo exists),exists)
-	GPU_TAGS := nvml
-endif
-ifeq ($(shell ldconfig -p | grep -q libdcgm.so && echo exists),exists)
```

Comment:
@rootfs is it ok to remove the dcgm library check?

Reply:
this is removing the `GPU_TAGS := dcgm` meaning the dcgm and NVML GPU code will be built into Kepler by default. The device plugins/code check at runtime for the libraries and don't run if the library doesn't exist.

The only reason I kept the HABANA tag is because there are issues building habana if the library doesn't exist on the system because of their cgo dependency